library/oneview_network_set.py

```diff
@@ -118,19 +118,21 @@ class NetworkSetModule(OneViewModule):
     MSG_ALREADY_PRESENT = 'Network Set is already present.'
     MSG_ALREADY_ABSENT = 'Network Set is already absent.'
     MSG_ETHERNET_NETWORK_NOT_FOUND = 'Ethernet Network not found: '
+    MSG_CONNECTION_TEMPLATE_RESET = 'Network Set connection template was reset to the default.'
     RESOURCE_FACT_NAME = 'network_set'
 
     argument_spec = dict(
         state=dict(
             required=True,
-            choices=['present', 'absent']
+            choices=['present', 'absent', 'default_bandwidth_reset']
```

Comment:
You have added 'default_bandwidth_reset' in choices but you are not using that anywhere. 
What operation will be performed if user sends 'default_bandwidth_reset' as a 'state'..?

Reply:
I tried default_bandwidth_reset for Network set which will take bandwidth from default connection template and updates network set connection template URI but we cant reset NetworkSet connection Template URI with default bandwidth. So, I had removed the method but forgot to remove this choice. Removed now.